tabulator/__init__.py

```diff
@@ -3,10 +3,22 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+import io
+import os
+
+
+# General
 
 from .stream import Stream
 from . import exceptions
 
 # Deprecated
+
 from .topen import topen
 from .stream import Stream as Table
+
+# Version
+
+__version__ = io.open(
```

Comment:
But setup.py or __init__.py are part of the package right? If it's defined there it's available as part of the package